dump-format.mediawiki

```diff
@@ -0,0 +1,66 @@
+<pre>
+  NEP: TBD
+  Title: Unified Neo chain dump format
```

Comment:
Just a concern: we're going to have state dump format as far, so may be it's better to use "block dump" format instead of "chain dump" in order to avoid confusion? Although "chain dump" is how we name it from the beginning of Neo.

@roman-khimov

Reply:
Check it against CLI and docs. It's known as "chain dump", "block dump", "offline package", but likely some term is prevailing. "State dump" is different, but that doesn't intersect with others to me. We never call this one "state dump".